miio/curtain_babai.py

```diff
@@ -0,0 +1,69 @@
+from enum import Enum
+
+import click
+
+from .click_common import EnumType, command, format_output
+from .miot_device import MiotDevice
+
+_MAPPING = {
+    # Source
+    # https://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:curtain:0000A00C:babai-190812:1:0000C805
+    # Curtain
+    # "motor_control": {"siid": 2, "piid": 1},  # 0 - Pause, 1 - Open, 2 - Close
+    "current_position": {"siid": 2, "piid": 2},  # Range: [0, 100, 1]
+    "target_position": {"siid": 2, "piid": 4},  # Range: [0, 100, 1]
+    "mode": {"siid": 2, 'piid': 4},  # 0 - Normal,  1 - Reversal, 2 - Calibrate
+    # Info
+    "Device Manufacturer": {"siid": 1, 'piid': 1},
+    "Device Model": {"siid": 1, 'piid': 2},
+    "Device Serial Number": {"siid": 1, 'piid': 3},
+    "Current Firmware Version": {"siid": 1, 'piid': 4},
+}
+# Model: "OnViz Curtain Controller (Wi-Fi)"
+MODEL_CURTAIN_BABAI = "babai.curtain.190812"
+
+
+class MotorControl(Enum):
+    Pause = 0
+    Open = 1
+    Close = 2
+
+
+class Mode(Enum):
+    Normal = 0
+    Reverse = 1
+    Calibrate = 2
+
+
+class BabaiCurtainMiot(MiotDevice):
+    """Main class representing the babai.curtain.190812 curtain."""
+
+    mapping = _MAPPING
+
```

Comment:
There is no `status()` nor status container defined in this class for the properties defined in the mapping.  Could you add one (and include an example response to the docstring of the container like shown here https://github.com/rytilahti/python-miio/blob/master/miio/heater_miot.py#L47)?